portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/PublicationsPage.testcase

```diff
@@ -564,6 +564,72 @@ definition {
 		}
 	}
 
+	@description = "Assert that is possible to publish Publication with Asset Publisher configurations"
+	@priority = 3
+	test CanPublishWithAssetPublisherConfigurations {
+		task ("Create two widget pages and add to each page an Asset Publisher") {
+			for (var pageNumber : list "1,2") {
+				JSONLayout.addPublicLayout(
+					groupName = "Guest",
+					layoutName = "Widget Page Name ${pageNumber}");
+
+				JSONLayout.addWidgetToPublicLayout(
+					groupName = "Guest",
+					layoutName = "Widget Page Name ${pageNumber}",
+					widgetName = "Asset Publisher");
+			}
+		}
+
+		task ("Add new Publication and use it") {
+			JSONPublications.addPublication(publicationName = "Publication Name");
+
+			Publications.selectPublication(publicationName = "Publication Name");
+		}
+
+		task ("Go to Widget Page Name 1 and configure Asset Publisher") {
+			Navigator.gotoPage(pageName = "Widget Page Name 1");
+
+			AssetPublisherPortlet.configureManualAssetSelectionPG();
```

Comment:
Configure AP with something visible so we can assert that we were able to configure AP in the UI